package.json

```diff
@@ -6,6 +6,7 @@
   "main": "index.js",
   "typings": "index.d.ts",
   "scripts": {
+    "build": "npm run build_all",
```

Comment:
just rename buidl_all to build should be better

Reply:
done